_posts/0000-01-02-anuragjul84.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
```

Comment:
Click the Files Changed tab in this pull request
Edit your newly added file
Replace line 5 with something new
Scroll to the bottom and click Commit Changes
Note : Can't find the button to edit the file? It may look like a pencil, or it may look like three dots.